Add instructions on copying and aliasing pages to the editor's guide (#7068)
wagtail/wagtail: docs/editor_manual/copying_aliasing_existing_pages.rst

.. _copying-page-label:

Copying Pages
_____________

Sometimes you might not want to create a new page from scrach. You could have several pages that are very similar to each other in terms of structure, but the content could differ slightly. In that case, you could copy an existing page and only change the required parts.

Here is how you can copy an existing page:

* When you hover over a page in an :doc:`Explorer page <finding_your_way_around/the_explorer_page>`, click on *More* and then *Copy*.

.. image:: ../_static/images/screen12.6_1_copy_from_explorer_menu.png
   :alt: Copy action available when hovering over a page in an explorer page

* You are then taken to a new page where you can input the new information including the new title, new slug and choosing the parent page (if you want to change it). Please note that you can also publish the copied page right away and crate an alias (please see :ref:`aliasing-page-label` for more information about aliasing pages).

.. image:: ../_static/images/screen12.6_2_copy_input_information.png
   :alt: Copy page form with the options to change the title, slug, parent page, published status and option to create an alias.

> * Once you change the information required, press *Copy this page* and the page will be successfully copied and appear in the Explorer page.

Reviewer: ```suggestion
* Once you completed this form, press *Copy this page*. The page will then be copied and appear in the Explorer page.
```

.. image:: ../_static/images/screen12.6_3_copy_success.png

>    :alt: Successful copying of the page resulting in it being available in an explorer page.

Reviewer: ```suggestion
   :alt: Successful copying of a page results in it being available in the Explorer page.
```

.. _aliasing-page-label:

Aliasing pages
______________

> When copying a page, you also have the option to create an alias to it. Let's say you have a page about Brioche in the Breads section. At the same time, you might also want to have it in the Pastries section. You could create a copy and change both whenever there is a modification to be made. A more efficient way that ensures synchronisation between the two is aliasing the page during the copy process.

Reviewer: ```suggestion
When copying a page, you also have the option to mark it as an alias. The content of an aliased page will stay in sync with the original. This is particularly useful when a page is required to available in multiple places.

For example, let's say you have a page about Brioche in the Breads section. You then want to also make this page available in the Pastries section. One way to do this is to create a copy of the Brioche page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias Wagtail will do this for you each time a modification to the original page is published.
```

Author: ```suggestion
When copying a page, you also have the option to mark it as an alias. The content of an aliased page will stay in sync with the original. This is particularly useful when a page is required to be available in multiple places.
For example, let's say you have a page about Brioche in the Breads section. You then want to also make this page available in the Pastries section. One way to do this is to create a copy of the Brioche page and change the parent page to the Pastries page. However you now need to remember to update this copy each time the original is modified. If a copy is marked as an alias Wagtail will do this for you each time a modification to the original page is published.
```

> Here is how you can create an alias to an existing page:

Reviewer: ```suggestion
Here is how to create an alias to an existing page:
```

> * The first step is the same as it was for :ref:`copying-page-label`. When you hover over a page in an :doc:`Explorer page <finding_your_way_around/the_explorer_page>`, click on *More* and then *Copy*.

Reviewer: ```suggestion
* The first step is the same as it is for :ref:`copying-page-label`. When you hover over a page in the :doc:`Explorer page <finding_your_way_around/the_explorer_page>`, click on *More* and then *Copy*.
```

* When you get to the copy page form, you can then choose another page as the parent page. Click the *Choose another page* button.

.. image:: ../_static/images/screen12.7_1_alias_choose_parent_page_button.png

>    :alt: Clicking the choose another page button during the copy page form in orderto change the parent of the copied page.

Reviewer: ```suggestion
   :alt: Clicking the choose another page button during the copy page form in order to change the parent of the copied page.
```

> * Select and click on the new page to act as the parent page.

Reviewer: ```suggestion
* Select and click on the desired parent page.
```

.. image:: ../_static/images/screen12.7_2_alias_choose_new_parent_page.png

>    :alt: Selecting and clicking on a new page to act as the new parent page.

Reviewer: ```suggestion
   :alt: Selecting and clicking the desired parent page.
```

> * Make sure the Alias checkbox is ticked. You can then complete the page aliasing by clicking on the *Copy this page* button.

Reviewer: ```suggestion
* Make sure the *Alias* checkbox is ticked. 
* You can then complete the page aliasing by clicking on the *Copy this page* button.
```

.. image:: ../_static/images/screen12.7_3_alias_confirm_changes.png
   :alt: Clicking on the Copy this page button to confirm aliasing.

> * The aliased page will then appear under the new parent page.

Reviewer: ```suggestion
* The aliased page will then appear under the new parent in the Explorer.
```

.. image:: ../_static/images/screen12.7_3_alias_page_new_parent.png

>    :alt: The aliased page appearing under the new parent page.

Reviewer: ```suggestion
   :alt: The aliased page appearing under the new parent page in the Explorer page.
```

> * If you try to edit the aliased page, you will get notified that it is an alias of another page and be offered two options: either to edit the original page (with changes appearing in both places) or convert the alias page into a regular page (i.e. a regular copy).

Reviewer: ```suggestion
* When you try to edit the aliased page, you are notified that it is an alias of another page. You will then be offered two options: edit the original page (with changes appearing in both places) or convert the alias page into a regular page (i.e. a regular copy, not updated when the original changes).
```

.. image:: ../_static/images/screen12.7_4_alias_page_edit_notification.png

>    :alt: Editing an alias page notifies it is an alias and offers to either edit the original page or change to a regular copy.

Reviewer: ```suggestion
   :alt: Editing an alias page notifies it is an alias and offers to either edit the original page or change the alias to a regular page.
```
